sys/net/gnrc/routing/rpl/gnrc_rpl_control_messages.c

```diff
@@ -419,8 +419,7 @@ bool _parse_options(int msg_type, gnrc_rpl_instance_t *inst, gnrc_rpl_opt_t *opt
                 dodag->dio_opts |= GNRC_RPL_REQ_DIO_OPT_PREFIX_INFO;
 #endif
                 gnrc_rpl_opt_prefix_info_t *pi = (gnrc_rpl_opt_prefix_info_t *) opt;
-                ipv6_addr_t all_RPL_nodes = GNRC_RPL_ALL_NODES_ADDR;
-                if_id = gnrc_ipv6_netif_find_by_addr(NULL, &all_RPL_nodes);
+                if_id = gnrc_ipv6_netif_find_by_addr(NULL, &ipv6_addr_all_rpl_nodes);
```

Comment:
Yea that's right. I am currently working on a PR that gets the right interface id from the netif header and passing that to the dodag. I am debugging that, though, something broke :/